README.md

```diff
@@ -1,3 +1,42 @@
 [![Build Status](https://travis-ci.org/agrc-widgets/layer-selector.svg)](https://travis-ci.org/agrc-widgets/layer-selector)
 [![Bower version](https://badge.fury.io/bo/agrc-layer-selector.svg)](https://badge.fury.io/bo/agrc-layer-selector)
 [![Sauce Test Status](https://saucelabs.com/browser-matrix/agrc-layerselector.svg)](https://saucelabs.com/u/agrc-layerselector)
+
+
+![layer-selector](/layer-selector.gif)
+
+## Usage
+
+ - [ArcGISTiledMapServiceLayers](./tests/example-mapserv-basemaps.html)
+ - [esri basemap](./tests/example-placement-over-esri-map.html)
+ - [Happy path Web Mercator](./tests/example-happy-path-tokens.html) * _requires quad token auth_
+ - [Linked Overlays](./tests/example-linked-overlays.html)
+ - [Custom LODS](./tests/example-custom-lods.html)
+
+## Development
+
+`grunt` - _Good for hot starts when your browser is already open_
```

Comment:
Looks like the numbered bullets are not indenting below this line. Maybe it needs a few spaces on the end?